packages/faency/package.json

```diff
@@ -36,7 +36,7 @@
     "prepublishOnly": "npm run build",
     "deploy-storybook": "build-storybook",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch --no-cache"
```

Comment:
@matthieuh thoughts on this?

Reply:
@paulocfjunior all PR changes looks great but this one is not solving any issue and will just makes test to run slower.

Reply:
Sure, I agree, I'll remove it